examples/applications/plot_tomography_l1_reconstruction.py

```diff
@@ -101,7 +101,7 @@ def generate_synthetic_data():
     rs = np.random.RandomState(0)
     n_pts = 36
     x, y = np.ogrid[0:l, 0:l]
-    mask_outer = (x - l / 2) ** 2 + (y - l / 2) ** 2 < (l / 2) ** 2
+    mask_outer = (x - l / 2.0) ** 2 + (y - l / 2.0) ** 2 < (l / 2.0) ** 2
```

Comment:
I don't think appending `.` after variable works. Anyways, I have removed the extra zeroes.